game-headed/build.gradle

```diff
@@ -11,18 +11,27 @@ mainClassName = 'org.triplea.game.headed.runner.HeadedGameRunner'
 version = getEngineVersion()
 
 ext {
+    javaFxRuntimeUrl = 'https://github.com/triplea-game/assets/raw/master/javafx/jfxrt-1.8.0_181.jar'
     releasesDir = file("$buildDir/releases")
     testFxVersion = '4.0.13-alpha'
 }
 
 dependencies {
     compile project(':game-core')
 
+    if (JavaVersion.current() >= JavaVersion.VERSION_1_9) {
+        compileOnly remoteLib(javaFxRuntimeUrl)
+    }
+
     testCompile project(':test-common')
     testCompile "org.sonatype.goodies:goodies-prefs:$sonatypeGoodiesPrefsVersion"
     testCompile "org.testfx:testfx-core:$testFxVersion"
     testCompile "org.testfx:testfx-junit5:$testFxVersion"
 
+    if (JavaVersion.current() >= JavaVersion.VERSION_1_9) {
+        testCompileOnly remoteLib(javaFxRuntimeUrl)
```

Comment:
Unfortunately, the `testCompileOnly` configuration does not inherit from the `compileOnly` configuration.  Research shows this appears to be a design decision by the Gradle maintainers (to be different than how Maven works with its `provideCompile` configuration).